f3probe.c

```diff
@@ -508,6 +508,10 @@ static int test_device(struct args *args)
 	}
 
 	free((void *)final_dev_filename);
+	if (fake_type != FKTY_GOOD) {
+		return 100 + fake_type;
+	}
+
```

Comment:
Drop the lines above and modify the `return` below as follows:
```
return fake_type == FKTY_GOOD ? 0 : 100 + fake_type;
```

Reply:
done